LISISO/upgrade.sh

```diff
@@ -67,6 +67,20 @@ if [ $distro_version = "unknown" ]; then
     exit 1
 fi
 
+OracleDistroName=$(grep -ihs "Orac" /etc/oracle-release)
+OracleDistroVersion=[52,53,54,55,56,57,58,59,510,511,60,61,62,63]
+case $OracleDistroName in
```

Comment:
same as above for upgrade

Reply:
Done

Logs:
[root@localhost LISISO]# ./upgrade.sh
raji 59
Upgrade is not supoorted for Oracle distros frpm 52 to 63